frontend/packages/dsco/package.json

```diff
@@ -17,6 +17,19 @@
   "license": "SEE LICENSE IN LICENSE",
   "sideEffects": false,
   "type": "module",
+  "exports": {
```

Comment:
Do components get added here automatically?

Reply:
No, I wanted to do that originally with the `clean-package` approach but had to delete it due to the way we're doing yarn workspace linking (the approach I was going with only worked in the case we did a publish/pull model but it appears we are not going to be using that model internally). This means each time we add a new component, we have to add an export here. 

If this becomes problematic, we will need to see if there's a way to update this programmatically.

Let me add a section on README on how to create new components to clarify this.